src/Filters/Actions/RedirectFilterAction.php

```diff
@@ -29,10 +29,8 @@ public function parse() {
             $this->require[] = 'copy';
             $script .= " :copy";
         }
-        if (isset($this->params['notify']) || isset($this->params['notify'])) {
+        if (isset($this->params['notify'])) {
```

Comment:
Can be better to test with empty as someone can passe something like `['address' => 'abc@example.com', 'notify' => false]` which shouldn't be interpreted as a condition to add notify

Reply:
Done